src/Sulu/Bundle/PageBundle/Controller/PageController.php

```diff
@@ -233,16 +233,18 @@ public function postTriggerAction(Request $request, string $id): Response
                     break;
                 case 'copy-locale':
                     $srcLocale = $this->getRequestParameter($request, 'src', false, $locale);
-                    $destLocale = $this->getRequestParameter($request, 'dest', true);
-                    $webspace = $this->getWebspace($request);
+                    $destLocales = $this->getRequestParameter($request, 'dest', true);
+                    $destLocales = \explode(',', $destLocales);
 
-                    // call repository method
-                    $data = $this->nodeRepository->copyLocale($id, $userId, $webspace, $srcLocale, \explode(',', $destLocale));
+                    $document = $this->documentManager->find($id, $srcLocale);
 
-                    if ($srcLocale !== $locale) {
-                        $data = $this->nodeRepository->getNode($id, $webspace, $locale);
-                    }
+                    $this->documentManager->copyLocale($document, $srcLocale, $destLocales);
```

Comment:
not sure if we should pass a single destination locale here. as far as i see, we do not pass an array of locales anywhere else. what do you think?

Reply:
maybe related to this: should there be a domain event for each destination locale?

Reply:
> not sure if we should pass a single destination locale here. as far as i see, we do not pass an array of locales anywhere else. what do you think?

fine for me, I don't have a strong opinion about that

> maybe related to this: should there be a domain event for each destination locale?

I would say yes, but that doesn't really affect the above decision IMO, because we could just loop over the locales and throw a specific domain event for each locale without any effort

/cc @sulu/core-team

Reply:
I have no strong opinion here both using a single local is okay from my side.